src/File.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace React\Http;
+
+use React\Stream\ReadableStreamInterface;
+
+class File implements FileInterface
+{
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @var string
+     */
+    protected $filename;
+
+    /**
+     * @var string
+     */
+    protected $type;
+
+    /**
+     * @var ReadableStreamInterface
+     */
+    protected $stream;
+
+    /**
+     * @param string $name
+     * @param string $filename
+     * @param string $type
+     * @param ReadableStreamInterface $stream
+     */
+    public function __construct($name, $filename, $type, ReadableStreamInterface $stream)
+    {
+        $this->name = $name;
+        $this->filename = $filename;
+        $this->type = $type;
+        $this->stream = $stream;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
```

Comment:
This is how they are called in multipart headers, I'd say this is expected naming.